locust/static/locust.js

```diff
@@ -104,6 +104,19 @@ $('#edit_form').submit(function(event) {
     );
 });
 
+String.prototype.toHHMMSS = function () {
```

Comment:
how about moving the js script into other single file?
So it will be easier to be maintained

Reply:
moved to timer.js